libs/status/type-defs/status-payloads/error-info.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @todo
+ */
+export type ErrorInfo = {
+  /**
+   * The reason of the error. This is a constant value that identifies the proximate cause of
+   * the error. Error reasons are unique within a particular domain of errors. This should be at
+   * most 63 characters and match a regular expression of `[A-Z][A-Z0-9_]+[A-Z0-9]`, which
```

Comment:
How come the max here in the comment is `63` when it's `64` for keys of `metadata`?

Reply:
Good question, I do not know why 😅, we can change it, I assume Google did it for some reason though